frontend/connection/src/utils.ts

```diff
@@ -25,10 +25,6 @@ const INITIAL_SLASH_RE = /^\/+/
 export function parseUriIntoBaseParts(url?: string): URL {
   const currentUrl = new URL(url ?? window.location.href)
 
-  if (!currentUrl.port) {
-    currentUrl.port = currentUrl.protocol === "https:" ? "443" : "80"
-  }
```

Comment:
I have removed these lines because they interestingly don't do anything. Try this in the dev console:

```javascript
const u = new URL("http://foo.bar/hello")
u.port = "80"
console.log(u.port)
```

You will see that the port is empty. The reason seems to be that the setter / getter does not set the value when it matches the default port for the respective protocol. When you set it to anything other than `"80"`, it will show up.